tests/bot/cogs/moderation/test_utils.py

```diff
@@ -0,0 +1,378 @@
+import unittest
+from datetime import datetime
+from unittest.mock import AsyncMock, MagicMock, call, patch
+
+from discord import Embed, Forbidden, HTTPException, NotFound
+
+from bot.api import ResponseCodeError
+from bot.cogs.moderation import utils
+from bot.constants import Colours, Icons
+from tests.helpers import MockBot, MockContext, MockMember, MockUser
+
+APPEAL_EMAIL = "appeals@pythondiscord.com"
+
+INFRACTION_TITLE = f"Please review our rules over at {utils.RULES_URL}"
+INFRACTION_APPEAL_FOOTER = f"To appeal this infraction, send an e-mail to {APPEAL_EMAIL}"
+INFRACTION_AUTHOR_NAME = "Infraction information"
+INFRACTION_COLOR = Colours.soft_red
+
+INFRACTION_DESCRIPTION_TEMPLATE = (
+    "\n**Type:** {type}\n"
+    "**Expires:** {expires}\n"
+    "**Reason:** {reason}\n"
+)
```

Comment:
As with the other constants, this could be defined and used by the utils module too.